remove-k8s-deployment.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
You don't need such a file. 
You can deploy with the k8s-deployment.yaml and delete also. 
`kubectl apply/delete -f k8s-deployment.yaml`

Reply:
uh, didn't know that. That's great!